collections/_article/godot-xr-update-nov-2025.md

```diff
@@ -0,0 +1,112 @@
+
+<iframe width="560" height="315" style="width: 100%; height: 100%; aspect-ratio: 16/9;" src="https://www.youtube.com/embed/L41KnIcSRzE" title="Things You Can Do With Two Hands" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+This game might have a simple aesthetic, but it is a unique experience to play through.
+
+The aim is to coordinate and perform actions with both your hands to complete levels. Whether it is turning keys in moving locks, throwing bowling balls that have to hit a cone at the same time, or trying to hit the moon while also hitting a target, it is deceptively tricky to complete the whole game. 
+
+### No 1. "Wiener Wobble Dog Park" by Andyman404
+
+[Wiener Wobble Dog Park](https://itch.io/jam/godot-xr-game-jam-sep-2025/rate/3883622) by [Andyman404](https://andyman404.itch.io/) ranked 1st with 7 ratings (Score: 4.048).
+
+<iframe width="560" height="315" style="width: 100%; height: 100%; aspect-ratio: 16/9;" src="https://www.youtube.com/embed/DNN8vEwrVdg" title="Wiener Wobble Dog Park" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+The winner of the game jam was Wiener Wobble Dog Park. This unique entry has you control a sausage dog with two fingers on each hand. One hand controls the front of the dog, while the other controls the rear.
+
+The dog works like a slinky and you have to move it through an obstacle course where you can move only when the dog touches the course. It is a frantic and very enjoyable experience.
+
+## Introducing spatial entities
+
+Spatial entities is a new collection of APIs within the OpenXR specification that gives us access to information about the real world around us. This API is designed to grow over time. Its initial iteration gives access to:
+
+1. Spatial anchors allow users to mark real world locations that are tracked by the headset. These can be stored on the headset and retrieved on subsequent runs of your application. They are great for placing virtual objects in a fixed location in your real space.
+2. Plane tracking provides us with insight about the world around us, where is our floor, where are walls, where are tables, etc.
+3. Marker tracking allows us to use various printed markers like QR codes, April tags, and Aruca codes, and have our headset identify and track those locations.
+
+Godot is an early adopter of this API thanks to funding through the Khronos Godot Integration Project.
+
+It is important to note that we are still awaiting public implementations of this API from various headset manufacturers and that not all headsets may implement the full API.
+We hope to share more about this exciting new API soon.
```

Comment:
@BastiaanOlij We probably want to update this and mention that Android XR supports the API.